gnomad/utils/vcf.py

```diff
@@ -297,31 +343,56 @@
             " deleterious."
         ),
     },
-    "revel_score": {
+    "revel_max": {
         "Number": "1",
         "Description": (
-            "dbNSFP's Revel score from 0 to 1. Variants with higher scores are"
+            "The maximum dbNSFP's Revel score at a site's Mane SELECT or canonical"
+            " transcript. Scores ranges from 0 to 1. Variants with higher scores are"
             " predicted to be more likely to be deleterious."
         ),
     },
-    "splice_ai_max_ds": {
+    "spliceai_ds_max": {
         "Number": "1",
         "Description": (
             "Illumina's SpliceAI max delta score; interpreted as the probability of the"
             " variant being splice-altering."
         ),
     },
-    "splice_ai_consequence": {
+    "pangolin_largest_ds": {
+        "Number": "1",
         "Description": (
-            "The consequence term associated with the max delta score in"
-            " 'splice_ai_max_ds'."
+            "Pangolin's largest delta score across 2 splicing consequences, which"
+            " reflects the probability of the variant being splice-altering"
         ),
     },
-    "primate_ai_score": {
+    "phylop": {
         "Number": "1",
         "Description": (
-            "PrimateAI's deleteriousness score from 0 (less deleterious) to 1 (more"
-            " deleterious)."
+            "Base-wise conservation score across the 241 placental mammals in the"
+            " Zoonomia project. Score ranges from -20 to 9.28, and reflects"
+            " acceleration (faster evolution than expected under neutral drift,"
+            " assigned negative scores) as well as conservation (slower than expected"
+            " evolution, assigned positive scores)."
+        ),
+    },
+    "sift_max": {
+        "Number": "1",
+        "Description": (
+            "Score reflecting the scaled probability of the amino acid substitution"
+            " being tolerated, ranging from 0 to 1. Scores below 0.05 are predicted to"
+            " impact protein function. We prioritize max scores for MANE Select"
```

Comment:
are we saying "MANE Select" or "MANE SELECT" (like in the description for revel_max?)

Reply:
Oops should be MANE Select I think